addon/components/ember-table-row.js

```diff
@@ -77,12 +77,12 @@ export default class EmberTableRow extends Component {
 
   click(event) {
     const tableObject = this.get('row.api.targetObject');
-    tableObject.send('onRowClicked', event, this.get('rowIndex'));
+    tableObject.send('onClick', event, this.get('rowIndex'));
```

Comment:
This is different because we are not sending this action externally. Really, if the table were using the row component directly, it would look like:

```
{{#ember-table as |row|}}
  {{ember-table-row onClick="onRowClicked"}}
{{/ember-table}}
```
So this action and the action on the table should remain `onRowClicked`